src/Aspire.Hosting.Yarp/ConfigurationBuilder/IYarpConfigurationBuilder.cs

```diff
@@ -46,17 +46,17 @@ public interface IYarpConfigurationBuilder
     /// <param name="clusterName">The name of the cluster.</param>
     /// <param name="destinations">The destinations used by this cluster.</param>
     /// <returns></returns>
-    public YarpCluster AddCluster(string clusterName, string[] destinations);
+    public YarpCluster AddCluster(string clusterName, object[] destinations);
```

Comment:
Do we need to do any additional type checking to see if `destinations` is one of a pre-defined set of things 
(string or EndpointReference or whatever else)?